src/icons/Settings/Settings.native.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
```

Comment:
Shouldn't this be called `Gear` Icon ?

Reply:
It's `settings` on figma. Even I was confused. Should I change it to gear?